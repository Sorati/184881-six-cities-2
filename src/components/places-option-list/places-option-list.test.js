import renderer from 'react-test-renderer';
import PlacesOptionList from './places-option-list';

it(`Places option list correctly renders with data`, () => {
  const tree = renderer
    .create(<PlacesOptionList
      isOpen={true}
      availableSorts={[`Popular`, `New`]}
      onItemClick={jest.fn()}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
