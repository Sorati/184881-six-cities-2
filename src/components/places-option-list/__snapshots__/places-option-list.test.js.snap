// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places option list correctly renders with data 1`] = `
<ul
  className="places__options places__options--custom places__options--opened"
>
  <li
    className="places__option places__option--active"
    onClick={[Function]}
    tabIndex="0"
  >
    Popular
  </li>
  <li
    className="places__option false"
    onClick={[Function]}
    tabIndex="0"
  >
    New
  </li>
</ul>
`;
